holidays/calendars/hindu.py

```
@@ -182,7 +183,44 @@ class _HinduLunisolar:
+        2025: (OCT, 22),
+        2026: (NOV, 10),
+        2027: (OCT, 30),
+        2028: (OCT, 18),
+        2029: (NOV, 6),
+        2030: (OCT, 27),
+        2031: (NOV, 15),
+        2032: (NOV, 3),
+        2033: (OCT, 23),
+        2034: (NOV, 13),
+        2035: (OCT, 31),
+    }
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)